app/include_upgrade_testnet.go

```diff
@@ -13,12 +13,14 @@ import (
 	v2 "github.com/babylonlabs-io/babylon/v4/app/upgrades/v2"
 	v22 "github.com/babylonlabs-io/babylon/v4/app/upgrades/v2_2"
 	v2rc4 "github.com/babylonlabs-io/babylon/v4/app/upgrades/v2rc4/testnet"
+	v4 "github.com/babylonlabs-io/babylon/v4/app/upgrades/v4"
 )
 
 // init is used to include v1 upgrade testnet data
 // it is also used for e2e testing
 func init() {
 	Upgrades = []upgrades.Upgrade{
+		v4.Upgrade,
 		v2rc4.Upgrade,
```

Comment:
I guess given that we inted to modify testnet to match mainnet, we will need to add
```go
		v23.Upgrade, // same as v3rc3 testnet
		v22.Upgrade,
```
upgrades here. Wydt @jhBharvest @RafilxTenfen @GAtom22 ?

Reply:
yes, we'll need to add these at some point. Either in this PR or a separated one